modules/claims_api/app/controllers/claims_api/v2/veterans/claims_controller.rb

```diff
@@ -168,6 +173,49 @@ def build_claim_structure(data:, lighthouse_id:, upstream_id:) # rubocop:disable
           }
         end
 
+        def get_phase_type_indicator_array(data)
+          return if data[:benefit_claim_details_dto][:phase_type_change_ind].nil?
+
+          (data[:benefit_claim_details_dto][:phase_type_change_ind]).split('')
+        end
+
+        def ever_phase_back(data)
+          return false if data[:benefit_claim_details_dto][:phase_type_change_ind].nil?
+
+          pt_ind_array = get_phase_type_indicator_array(data)
+          pt_ind_array.first.to_i > pt_ind_array.last.to_i
+        end
+
+        def get_evss_claim(data, phase_number)
+          ClaimsApi::BGSClaimStatusMapper.new(data[:benefit_claim_details_dto], phase_number).name_from_phase
+        end
```

Comment:
This method is pretty confusing.
It's called "get_evss_claim", but V2 isn't getting claims from EVSS.
The method also isn't actually getting a claim.
It's getting a name value.